Elevate_QA_API.xml

```diff
@@ -12,6 +12,14 @@
                 </methods>
             </class>
 
+            <class name="org.shikshalokam.backend.ep.TestUserRoleExtensionCreate">
+                <methods>
+                    <include name="testUserRoleCreateWithValidPayload"/>
+                    <include name="testUserRoleCreateWithInvalidPayload"/>
+                    <include name="testUserRoleCreateWithEmptyFields"/>
+                    <include name="createUserRoleExtension"/>
```

Comment:
createUserRoleExtension is not a test method .
here we need to include only  @Test methods only .